src/config.js

```diff
@@ -36,6 +36,12 @@ export const flags = {
     default: true,
     icon: '🐳',
   },
+  expandAllPipelines: {
+    name: 'Expand all modular pipelines',
+    description: 'Expand all modular pipelines on first load',
+    default: false,
+    icon: '🔛',
+  },
```

Comment:
I'm curious what the logic is for putting this here, in the `Experiments` section, versus displaying it in the `General` section of the setting panel?

Reply:
The `Experiments` section is where we put all our feature flags ( such as `sizeWarning`, and in this case, `expandAllPipelines`), while `prettyNaming` is not hidden behind a feature flag but rather as a general setting.

Reply:
Right, I see. From the readme: _"Kedro-Viz uses features flags to roll out some experimental features."_

Let me reword my question: is this `expandAllPipelines` behavior an experimental feature?

Reply:
yep it is 😄  experimental features are features where we don't officially roll out to all users but are accessible to power users for testing. For example, when we did the initial version of modular pipelines, we hid it behind a feature flag to allow it to still be accessible for users while we continue polishing the feature. 

In this case, the 'expandAllPipelines' feature is an experimental feature that we can make accessible to some users given their mention of this in an issue, where we can make this accessible officially as a feature when we have given it some design love in the future.